standard/expressions.md

```diff
@@ -3478,95 +3478,95 @@ The predefined remainder operators are listed below. The operators all compute t
-      <td><code>NaN</code></td>
-      <td><code>NaN</code></td>
-    </tr>
-  </table>
+<!-- md equivalent: ` `   | **`+y`**  | **`-y`**  | **`+0`**  | **`-0`**  | **`+∞`**  | **`-∞`**  | **`NaN`** -->
+  <tr>
+    <td></td>
+    <td><b><code>+y</code></b></td>
+    <td><b><code>-y</code></b></td>
+    <td><b><code>+0</code></b></td>
+    <td><b><code>-0</code></b></td>
+    <td><b><code>+∞</code></b></td>
+    <td><b><code>–∞</code></b></td>
+    <td><b><code>NaN</code></b></td>
+  </tr>
+<!-- md equivalent: `+x`  | `+z`  | `+z`  | `NaN` | `NaN` | `+x`  | `+x`  | `NaN` -->
+  <tr>
+    <td><code>+x</code></td>
+    <td><code>+z</code></td>
+    <td><code>+z</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>+x</code></td>
+    <td><code>+x</code></td>
+    <td><code>NaN</code></td>
+  </tr>
+<!-- md equivalent: `-x`  | `-z`  | `-z`  | `NaN` | `NaN` | `-x`  | `-x`  | `NaN` -->
+  <tr>
+    <td><code>-x</code></td>
+    <td><code>-z</code></td>
+    <td><code>-z</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>-x</code></td>
+    <td><code>-x</code></td>
+    <td><code>NaN</code></td>
+  </tr>
+<!-- md equivalent: `+0`  | `+0`  | `+0`  | `NaN` | `NaN` | `+0`  | `+0`  | `NaN` -->
+  <tr>
+    <td><code>+0</code></td>
+    <td><code>+0</code></td>
+    <td><code>+0</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>+0</code></td>
+    <td><code>+0</code></td>
+    <td><code>NaN</code></td>
+  </tr>
+<!-- md equivalent: `-0`  | `-0`  | `-0`  | `NaN` | `NaN` | `-0`  | `-0`  | `NaN` -->
+  <tr>
+    <td><code>-0</code></td>
+    <td><code>-0</code></td>
+    <td><code>-0</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>-0</code></td>
+    <td><code>-0</code></td>
+    <td><code>NaN</code></td>
+  </tr>
+<!-- md equivalent: `+∞`  | `NaN` | `NaN` | `NaN` | `NaN` | `NaN` | `NaN` | `NaN` -->
+  <tr>
+    <td><code>+∞</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+  </tr>
+<!-- md equivalent: `-∞`  | `NaN` | `NaN` | `NaN` | `NaN` | `NaN` | `NaN` | `NaN` -->
+  <tr>
+    <td><code>-∞</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+  </tr>
+<!-- md equivalent: `NaN` | `NaN` | `NaN` | `NaN` | `NaN` | `NaN` | `NaN` | `NaN` -->
+  <tr>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+    <td><code>NaN</code></td>
+  </tr>
+</table>
 - Decimal remainder:
```

Comment:
I believe if we add a blank line after the table, the Word converter should be happy.